tests/system/marathon_pods_tests.py

```diff
@@ -595,12 +617,8 @@ def wait_for_pod_recovery():
     path2 = tasks[1]['container']['volumes'][0]['container_path']
     logger.info('Deployd two containers on {}:{}/{} and {}:{}/{}'.format(host, port1, path1, host, port2, path2))
 
-    @retrying.retry(wait_fixed=1000, stop_max_attempt_number=30, retry_on_exception=common.ignore_exception)
-    def check_data(port, path):
-        cmd = "curl {}:{}/{}/foo".format(host, port, path)
-        run, data = run_command_on_master(cmd)
-        assert run, "{} did not succeed".format(cmd)
-        assert 'hello\nhello\n' in data, "'hello\nhello\n' not found in '{}'n".format(data)
+    expected_data2 = next((t['id'] for t in tasks if t['name'] == 'container1'), None)
```

Comment:
could you put this comment here as well? Or maybe extract that code?
```
 # Container with the name 'container1' appends its taskId to the file. So we search for the
    # taskId of that container which is not always the tasks[0]
```

Reply:
I mean, this same is used in this method a [few lines above](https://github.com/mesosphere/marathon/pull/6830/files/2753fa5d87b9eafd194d7390317095034094905b#diff-abc76c3195bf940d3a912a659eff1e3cR581) - I felt that it would be sufficient for the reader ;)

Reply:
But if you insist ;)